lib/mix/tasks/torch.install.ex

```diff
@@ -41,8 +41,8 @@ defmodule Mix.Tasks.Torch.Install do
     do: phoenix_version |> to_string() |> template_file(format)
 
   defp template_file(phoenix_version, format) when is_binary(phoenix_version) do
-    if Version.match?(phoenix_version, ">= 1.5.0") do
-      "layout.phx1_5.html.#{format}"
```

Comment:
This conditional logic (and related directory stuff) can/should be removed completely.  Since we only intend to support Phx 1.6+ going forward, that is the new "baseline" and any conditional logic around that should adjust to assume 1.6.

If you want that to come as a future PR, jus create an Issue for it and link it in a commen to this PR.